docs/book/src/quinn/certificate.md

```diff
@@ -0,0 +1,163 @@
+# Certificates
+
+In this chapter, we discuss the configuration of the certificates that is **required** for a working Quinn connection. 
+
+QUIC uses TLS 1.3 for authentication of connections, the server will have to provide the client with a certificate confirming its identity, 
+and the customer must be configured to trust the certificates he receives from our server. 
+
+## Insecure Connection
+
+For our example use case, the easiest way to allow the client to trust our server is to disable certificate verification (don't do this in production!). 
+When the `dangerous_configuration` feature flag of [rustls][3] is enabled, a client can be configured to trust any server.
+
+Start with adding a [rustls][3] dependency with the `dangerous_configuration` feature flag to your `Cargo.toml` file.
+
+```toml
+quinn = "*"
+rustls = { version = "*", features = ["dangerous_configuration", "quic"] }
+``` 
+
+Then, you can skip the certificate validation on the client by implementing [ServerCertVerifier][ServerCertVerifier] and let it assert true for any server. 
+
+```rust
+// Implementation of `ServerCertVerifier` that verifies everything as trustworthy.
+struct SkipCertificationVerification;
+
+impl rustls::ServerCertVerifier for SkipCertificationVerification {
+    fn verify_server_cert(
+        &self, _: &rustls::RootCertStore, _: &[rustls::Certificate], _: webpki::DNSNameRef, _: &[u8],
+    ) -> Result<rustls::ServerCertVerified, rustls::TLSError> {
+        Ok(ServerCertVerified::assertion())
+    }
+}
+```
+
+After that, we can configure our [ClientConfig][ClientConfig] to use this new [ServerCertVerifier][ServerCertVerifier]. 
+
+```rust
+pub fn insecure() -> ClientConfig {
+    let mut cfg = quinn::ClientConfigBuilder::default().build();
+
+    // Get a mutable reference to the 'crypto' config in the 'client config'..
+    let tls_cfg: &mut rustls::ClientConfig =
+        std::sync::Arc::get_mut(&mut cfg.crypto).unwrap();
+
+    // Change the certification verifier.
+    // This is only available when compiled with 'dangerous_configuration' feature.
+    tls_cfg
+        .dangerous()
+        .set_certificate_verifier(Arc::new(SkipCertificationVerification));
+    cfg
+}
+```
+ 
+Finally, if you plug this [ClientConfig][ClientConfig] into the [EndpointBuilder::default_client_config()][default_client_config] your client endpoint should verify all connections as trustworthy.
+
+## Using Certificates
+
+In this section we look at certifying an endpoint with a real certificate. 
+This can be done with either a real certificate or a self-identified certificate.
+
+### Self Signed
+
+A [self-signed][5] certificate entails that the user signing it will be its own CA. 
+These certificates are easy to create and cost no money. 
```

Comment:
```suggestion
Relying on [self-signed][5] certificates means that clients allow servers to sign their own certificates. This is simpler because no third party has to be involved in signing the server's certificate.
```